lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+
+# MSG_PRINT_FAN c=10
+#: ultralcd.cpp:1447
+msgid "Print FAN"
+msgstr "Utskrfl\xe1kt"
+
+# MSG_WIZARD_LOAD_FILAMENT c=20 r=6
+#: ultralcd.cpp:4818
+msgid "Please insert filament into the extruder, then press the knob to load it."
+msgstr "V\xe1nligen s\xe1tt i filamentet i extrudern, tryck sedan pa knappen f\xefr inladdning.."
+
+# MSG_MMU_INSERT_FILAMENT_FIRST_TUBE c=20 r=6
+#: ultralcd.cpp:4813
+msgid "Please insert filament into the first tube of the MMU, then press the knob to load it."
+msgstr "V\xe1nligen s\xe1tt i filamentet i den f\xefrsta kanalen pa MMU:n och tryck sedan pa knappen f\xefr inladdning."
+
+# MSG_PLEASE_LOAD_PLA c=20 r=4
+#: ultralcd.cpp:4735
+msgid "Please load filament first."
+msgstr "V\xe1nligen ladda filament f\xefrst."
+
+# MSG_BED_CORRECTION_REAR c=14
+#: ultralcd.cpp:3117
+msgid "Rear side [um]"
+msgstr "Baksida [um]"
+
+# MSG_UNLOAD_FILAMENT_REPEAT c=20 r=4
+#: ultralcd.cpp:7404
+msgid "Please unload the filament first, then repeat this action."
+msgstr "V\xe1nligen mata ut filamentet f\xefrst och upprepa sedan denna atg\xe1rd."
+
+# MSG_CHECK_IR_CONNECTION c=20 r=4
+#: ultralcd.cpp:7407
+msgid "Please check the IR sensor connection, unload filament if present."
+msgstr "Kontrollera IR-sensorns anslutning, mata ut eventuellt filament."
+
+# MSG_RECOVERING_PRINT c=20
+#: Marlin_main.cpp:11393
+msgid "Recovering print"
+msgstr "Aterst\xe1ller utskrift"
+
+# MSG_REMOVE_OLD_FILAMENT c=20 r=5
+#: mmu.cpp:833
+msgid "Remove old filament and press the knob to start loading new filament."
+msgstr "Ta bort det gamla filamentet och tryck pa knappen f\xefr att b\xefrja ladda nytt filament."
+
+# MSG_CALIBRATE_BED_RESET c=18
+#: ultralcd.cpp:5804
+msgid "Reset XYZ calibr."
+msgstr "Aterst\xe1ll XYZ-kal."
+
+# MSG_RESET c=14
+#: messages.c:85
+msgid "Reset"
+msgstr "Aterst\xe1ll"
+
+# MSG_RESUME_PRINT c=18
+#: messages.c:86
+msgid "Resume print"
+msgstr "Ateruppta utskrift"
+
+# MSG_RESUMING_PRINT c=20
+#: messages.c:87
+msgid "Resuming print"
+msgstr "Aterupptar utskrift"
+
+# MSG_BED_CORRECTION_RIGHT c=14
+#: ultralcd.cpp:3115
+msgid "Right side[um]"
+msgstr "H\xefger sida[um]"
+
+# MSG_RPI_PORT c=13
+#: messages.c:142
+msgid "RPi port"
+msgstr "RPi port"
+
+# MSG_WIZARD_RERUN c=20 r=7
+#: ultralcd.cpp:4756
+msgid "Running Wizard will delete current calibration results and start from the beginning. Continue?"
+msgstr "Att utf\xefra guiden kommer att radera aktuella kalibreringsinst\xe1llningar och b\xefrja om fran b\xefrjan. Forts\xe1tta?"
+
+# MSG_SD_CARD c=8
+#: messages.c:138
+msgid "SD card"
+msgstr "SD-kort"
+
+# MSG_RIGHT c=10
+#: ultralcd.cpp:2845
+msgid "Right"
+msgstr "H\xefger"
+
+# MSG_FIND_BED_OFFSET_AND_SKEW_LINE1 c=20 r=3
+#: messages.c:44
+msgid "Searching bed calibration point"
+msgstr "S\xefker efter kalibreringspunkt f\xefr b\xe1dden"
+
+# MSG_LANGUAGE_SELECT c=18
+#: ultralcd.cpp:4451
+msgid "Select language"
+msgstr "V\xe1lj sprak"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.